### DISCUSSION TITLE
ZLUDA/blog/zluda-update-q4-2024/

### DISCUSSION THREAD

**Discussion by giscus** [vosen/zluda_website#2] [Announcements]
# ZLUDA/blog/zluda-update-q4-2024/

https://vosen.github.io/ZLUDA/blog/zluda-update-q4-2024/

**Comment by pervognsen** [comment]
Nice write-up. Regarding atomic increment-with-modulo, it could be useful for ring buffer read/write indices where the ring buffer has a non-power-of-two size. And it would be even useful (albeit less useful) in the power-of-two size case by letting you remove one 'and' instruction on the load/store path.

**Comment by Anazark** [comment]
I don't understand any of the technicalities but I'm rooting for you !!
Thank you for persevering in this project, I will pray for your success !

**Comment by BenHizak** [comment]
I learned a lot about intermediate languages. Thank you.
